amps/ags/rm-community/rm-community-repo/pom.xml

```diff
@@ -548,7 +548,16 @@
                    <configuration>
                       <images>
                          <image>
-                            <name>${image.name}:${image.tag}</name>
+                             <name>${image.name}:${image.tag}</name>
+                             <build>
+                                 <buildx>
+                                     <platforms>
+                                         <platform>linux/amd64</platform>
+                                         <platform>linux/arm64</platform>
+                                     </platforms>
+                                 </buildx>
+                                 <dockerFileDir>${project.basedir}</dockerFileDir>
```

Comment:
`dockerFileDir` is a deprecated configuration option that should be replaced with `contextDir` instead, relevant documentation: https://dmp.fabric8.io/#context-dir.

Furthermore, when overriding the `contextDir`, we should make sure to understand why this is necessary.

Reply:
Changed to contextDir. Based on documentation we need that parameter to use existing Dockerfile: Alternatively an external Dockerfile template or Docker archive can be used. This mode is switched on by using one of these three configuration options within: contextDir, dockerFile, dockerArchive, dockerFileDir (deprecated, use contextDir).